README.md

```diff
@@ -1,31 +1,63 @@
-Processing
-==========
-
-This is the official source code for the [Processing](http://processing.org) Development Environment (PDE), 
-the “core” and the libraries that are included with the [download](http://processing.org/download).
-
-__I've found a bug!__ 
-Let us know [here](https://github.com/processing/processing/issues) (after first checking if someone has already posted a similar problem). 
-If it's a documentation, web site, or examples problem, take that up with folks [here](https://github.com/processing/processing-docs/issues). 
-There are also separate locations for [Android Mode](https://github.com/processing/processing-android/issues), or the [Video](https://github.com/processing/processing-video/issues) and [Sound](https://github.com/processing/processing-sound/issues) libraries. 
-The [processing.js](http://processingjs.org) project is not affiliated with us, but you can find their issue tracker [here](https://github.com/processing-js/processing-js/issues).
-
-__That [processing-bugs](https://github.com/processing-bugs) fella is a damn liar.__ 
-The issues list has been imported from Google Code, so there are many spurious references 
-amongst them since the numbering changed. Basically, any time you see references to 
-changes made by [processing-bugs](https://github.com/processing-bugs), it may be somewhat suspect.
+# Processing
+
+This is the official source code for the [Processing][processing site] Development Environment (PDE)
+, the “core” and the libraries that are included with the [download][processing download].
+
+**I've found a bug!**
```

Comment:
@yukiisbored their standard seems to use `__` for bold text, not `**`, I'd recommend to configure markdownbear to their preferences. (Also applies for headings and probably more.)